graph-manager-docs/source/setup-analytics.md

```diff
@@ -107,29 +91,53 @@ query AuthorForPost {
 }
 ```
 
-Even though this concept lacks definition, it's important to decide on how queries should be grouped together when tracking metrics about GraphQL execution. The concept of a **"query signature"** is what we use at Apollo to group similar operations together even if their exact textual representations are not identical. The query signature, along with the operation name, are used to group queries together in the `FullTracesReport`.
+It's important to decide how to group such queries when tracking metrics. The [TypeScript reference implementation](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/signature.ts) does the following to every query before generating its signature to better group functionally equivalent operations:
+
+* Drop unused fragments and/or operations
+* Hide string literals
+* Ignore aliases
+* Sort the tree deterministically
+* Ignore differences in whitespace.
+
+We recommend using the same default signature method for consistency across different server runtimes.
+
+### Sending metrics to the reporting endpoint
+
+After your GraphQL server prepares a batch of traces, it should send them to the Graph Manager reporting endpoint, at the following URL:
+
+```
+https://engine-report.apollodata.com/api/ingress/traces
+```
+
+Each batch should be sent as an HTTP POST request. The body of the request can be one of the following:
+
+* A binary serialization of a `FullTracesReport` message
+* A _Gzipped_ binary serialization of a `FullTracesReport` message
+
+To authenticate with Graph Manager, each request must include either:
 
-The TypeScript reference implementation uses a default signature method and allows for that signature method to also be overridden by the user. The [implementation of the default](https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting/src/signature.ts) drops unused fragments and/or operations, hides String literals, ignores aliases, sorts the tree deterministically, and ignores whitespace differences. We recommend using the same default signature method for consistency across different server runtimes.
+* An `X-Api-Key` header with a valid API key for your graph
+* An `authtoken` cookie with a valid API key for your graph
```

Comment:
we should probably say only graph-level tokens `service:foo:25902u3r90j` are supported